phpunit.xml

```diff
@@ -11,10 +11,9 @@
 	</php>
 	<testsuites>
 		<testsuite>
-			<file>tests/phpunit/tests/two-factor.php</file>
-			<file>tests/phpunit/tests/providers/class.two-factor-dummy.php</file>
-			<!-- This test fails in PHP 5.2. FIDO U2F support is PHP 5.3+ only. -->
-			<file phpVersion="5.3.0" phpVersionOperator=">=">tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</file>
+			<directory suffix=".php">tests/phpunit/tests</directory>
+			<exclude>tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</exclude>
+			<file phpVersion="5.3.0">tests/phpunit/tests/providers/class.two-factor-fido-u2f.php</file>
```

Comment:
(re: excluding phpVersionOperator, I was borrowing from WordPress's phpunit.xml and stole that syntax)